openapi.go

```diff
@@ -19,7 +19,7 @@ func NewOpenAPI() *OpenAPI {
 	desc := NewOpenApiSpec()
 	return &OpenAPI{
 		description:            &desc,
-		generator:              openapi3gen.NewGenerator(),
+		generator:              openapi3gen.NewGenerator(openapi3gen.SchemaCustomizer(SchemaCustomizer)),
```

Comment:
🤔 we should just default to this in general and the provide a engine option to override it.